src/app/retrospective/sprint-task-summary/sprint-task-summary.component.ts

```diff
@@ -325,6 +327,12 @@ export class SprintTaskSummaryComponent implements OnInit, OnChanges, OnDestroy
                     newRowsAction: 'keep',
                     clearButton: true,
                 },
+                cellRenderer: (params) => {
+                    if (!params.data.IsTrackerTask || !params.data.URL) {
```

Comment:
could we use lodash here?

Reply:
There is no direct method in lodash to check for multiple keys simultaneously as of now, but there is one workaround for that, which is a little less readable. Refer https://stackoverflow.com/questions/29001762/lodash-has-for-multiple-keys